src/Database/PostgreSQL/Consumers/Utils.hs

```diff
@@ -73,3 +78,6 @@ forkImpl ffork tname m = E.mask $ \release -> do
       E.Handler $ \StopExecution -> return ()
     , E.Handler $ (throwTo parent . ThrownFrom tname)
     ]
+
+hashedName :: Text -> RawSQL () -> QueryName
+hashedName baseName tableName = QueryName . (mappend "consumers")  . pack . show . hash $ unRawSQL tableName <> baseName
```

Comment:
IIUC with this you won't know from postgresql logs for which table/operation the query is executed, I doubt @marco44 would like that :slightly_smiling_face: 

I propose to just do `take 63 $ baseName ++ "$" ++ tableName`, it's doubtful there will be two job or consumer tables with the same prefix of length ~50.

Reply:
Easy enough to fix.